src/pretix/base/models/event.py

```diff
@@ -481,7 +514,22 @@ def best_availability(self):
                 quota_used_for_possible_tickets[q] += possible_of_this
 
             best_state_found = max(best_state_found, worst_state_for_ticket)
-        return best_state_found, num_tickets_found, num_tickets_possible
+
+        waiting_list_found = False
+        if best_state_found < Quota.AVAILABILITY_OK:
+            for k, quota_list in quotas_for_item.items():
+                worst_state_for_ticket = min(r[q][0] for q in quota_list)
+                if worst_state_for_ticket < Quota.AVAILABILITY_OK and k in items_with_waiting_list:
+                    waiting_list_found = True
+                    break
+            else:
+                for k, quota_list in quotas_for_variation.items():
+                    worst_state_for_ticket = min(r[q][0] for q in quota_list)
+                    if worst_state_for_ticket < Quota.AVAILABILITY_OK and k in vars_with_waiting_list:
+                        waiting_list_found = True
+                        break
+
+        return best_state_found, num_tickets_found, num_tickets_possible, waiting_list_found
```

Comment:
Very nitkpicky: this always returns `waiting_list_found=False` for `best_state_found==Quota.AVAILABILITY_OK`. I know we do not need that info if the event is available, but it might be confusing lateron. Maybe instead return None, if we have not checked? And add a comment in the doc-string, that if `best_state_found==Quota.AVAILABILITY_OK` the returned value of `waiting_list_found` cannot be relied upon?